spec/controllers/api/v2/footnotes_controller_spec.rb

```diff
@@ -1,108 +1,147 @@
           },
-          {
-            id: String,
-            type: 'measure',
-            attributes: {
-              goods_nomenclature_item_id: String,
-              validity_start_date: String,
-              validity_end_date: String,
+          relationships: {
+            measures: {
+              data: [
+                {
+                  id: String,
+                  type: 'measure',
+                },
+              ],
             },
-            relationships: {
-              goods_nomenclature: {
-                data: {
+            goods_nomenclatures: {
+              data: [
+                {
                   id: String,
                   type: 'heading',
                 },
-              },
-              geographical_area: {
-                data: {
+                {
                   id: String,
-                  type: 'geographical_area',
+                  type: 'chapter',
                 },
-              },
+              ],
             },
           },
-        ],
-        meta: {
-          pagination: {
-            page: Integer,
-            per_page: Integer,
-            total_count: Integer,
+        },
+      ],
+      included: [
+        {
+          id: String,
+          type: 'chapter',
+          attributes: {
+            goods_nomenclature_item_id: String,
+            description: String,
+            formatted_description: String,
+            producline_suffix: String,
           },
         },
-      }
+        {
+          id: String,
+          type: 'heading',
+          attributes: {
+            goods_nomenclature_item_id: String,
+            description: String,
+            formatted_description: String,
+            producline_suffix: String,
+          },
+        },
+        {
+          id: String,
+          type: 'heading',
+          attributes: {
+            goods_nomenclature_item_id: String,
+            description: String,
+            formatted_description: String,
+            producline_suffix: String,
+          },
+        },
+        {
+          id: String,
+          type: 'measure',
+          attributes: {
+            goods_nomenclature_item_id: String,
+            validity_start_date: String,
+            validity_end_date: String,
+          },
+          relationships: {
+            goods_nomenclature: {
+              data: {
+                id: String,
+                type: 'heading',
+              },
+            },
+            geographical_area: {
+              data: {
+                id: String,
+                type: 'geographical_area',
+              },
+            },
+          },
+        },
+      ],
+      meta: {
+        pagination: {
+          page: Integer,
+          per_page: Integer,
+          total_count: Integer,
+        },
+      },
+    }
+  end
+
+  before do
+    measure_with_goods_nomenclature = create(:measure, :with_base_regulation, goods_nomenclature: create(:heading, :with_description))
```

Comment:
maybe it can help, there is a trait called 
`:with_goods_nomenclature`

Reply:
I wanted a heading, though

Reply:
We could add a trait :with_goods_nomenclature_heading but it seemed overkill